docs/source/markdown/podman-run.1.md

```diff
@@ -405,6 +405,14 @@ Meaning **groupname** is initially mapped to gid **100000** which is referenced
 above: The group **groupname** is mapped to group **100000** of the initial namespace then the
 **30000**st id of this namespace (which is gid 130000 in this namespace) is mapped to container namespace group id **0**. (groupname -> 100000 / 30000 -> 0)
 
+#### **--gpus**=*all|device=*
+
+Since containerd does not support executing prestart hooks, Docker introduced the `--gpu` flag which is a hardcoded call to the NVIDIA prestart hook.
```

Comment:
```suggestion
Since containerd does not support executing prestart hooks, Docker introduced the `--gpu` flag, which is a hardcoded call to the NVIDIA prestart hook.
```